107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -247,18 +264,10 @@ private static void addIdentityCopierIfNoneRegistered(Map<Class<?>, ClassInstanc
     if (cacheLoader == null && cacheWriter == null) {
       return null;
     } else {
-      return new Eh107CacheLoaderWriter<K, V>(cacheLoader, cacheWriter);
+      return new Eh107CacheLoaderWriter<K, V>(cacheLoader, config.isReadThrough(), cacheWriter, config.isWriteThrough());
     }
   }
 
-  @SuppressWarnings("unchecked")
-  private static <K, V> Factory<CacheWriter<K, V>> getCacheWriterFactory(CompleteConfiguration<K, V> config) {
-    // I could be wrong, but I don't think this factory should be typed the way it is. The factory
-    // should be parameterized with (K, V) and it's methods take <? extend K>, etc
```

Comment:
Not much we can do about it as it talks about a JSR-107 type thus external to us.